tests/integration/targets/ntnx_ndb_clusters/tasks/register_cluster.yml

```diff
@@ -0,0 +1,238 @@
+---
+- debug:
+    msg: Start testing ntnx_ndb_clusters
```

Comment:
Add test case to check the spec created using check_mode. This will help us to identify if correct spec is being sent to server